CRDI/Code/DIInjector/CRDIInjector.m

```diff
@@ -47,6 +49,8 @@ - (id)initWithContainer:(CRDIContainer *)aContainer
         self.container = aContainer;
         self.classInspector = [CRDIClassInspector new];
         self.classesCache = [NSMutableDictionary new];
+        self.queue = dispatch_queue_create("CRDIInjectorIsolation", 0);
```

Comment:
According to docs we should either pass queue attribute explicitly as `DISPATCH_QUEUE_SERIAL` or use `NULL`